tests/subsys/settings/nffs/raw/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(NONE)
 
 add_subdirectory(../src nffs_test_bindir)
+
```

Comment:
add_subdirectory(../../fs/src settings_test_fs_bindir)

Reply:
I do not think it is needed. raw only builds ../src, contents. ./src contains CMakeList that specifies this settings_test_fs_bindir.